README.md

```diff
@@ -100,8 +100,8 @@ In terms of the nix code, you should not have to leave [flake.nix](./flake.nix)
 
 That being said, if only for better understanding, there is a guide to going outside of those 2 files in [:help nixCats.flake.nixperts.nvimBuilder](./nix/nixCatsHelp/nvimBuilder.txt) in case you want to.
 
-All config folders like ftplugin and after work as designed (see :h rtp), if you want lazy loading put it in optionalPlugins in a category in the flake and call packadd when you want it.
-Although, it does specifically expect init.lua rather than init.vim at root level.
+All config folders like `ftplugin/` and `after/` work as designed (see :h rtp), if you want lazy loading put it in `optionalPlugins` in a category in the flake and call `packadd` when you want it.
```

Comment:
just marking the directories as such with code formatting and a trailing slash, and code formatting the attributes/commands.